src/schemas/space.json

```diff
@@ -221,6 +221,17 @@
               "collector"
             ]
           }
+        },
+        "parent": {
+          "type": "string",
+          "title": "parent"
+        },
+        "children": {
```

Comment:
Can you enforce a max number of item here? I would recommend 8. This is just to avoid someone adding 1k space there and overload GraphQL

Reply:
done. what about validation now? maybe it can just stay like that and we perform a check in the settings page. the best solution would probably be to add a new validator that checks spaces existences based on a hub url param or so.

Reply:
We dont need to validate that space exist here, but on the frontend we can make it that you can only select existing space.